### DISCUSSION TITLE
非常奇怪的问题：上传一段时间/一定量数据后，上传速度变得很慢

### DISCUSSION THREAD

**Discussion by gdtv** [AlistGo/alist#6917] [General]
我在服务器上同时安装了alist和rclone，挂载了以下存储：
A：alist挂载123云盘(驱动为123云盘)
B：alist挂载123云盘(驱动为webdav盘)
C：alist挂载移动云盘
D：alist挂载阿里云OSS(驱动为对象存储，不是阿里云网盘)
E：rclone通过webdav挂载了123云盘

遇到的问题：

**平常上传速度是100Mbps左右。每当我用A或者B方式连续上传了几十到100多G文件，或者是alist启动几小时到十几小时后，(我不知那个是触发条件)，A、B、C、E这4种方式的上传速度都变成了只有10Mbps左右，只有D方式还有100Mbps左右。
重启alist或者重启系统后A、B、C、E这4种方式的上传速度都恢复正常。**

怀疑是网盘限制，但是为什么重启alist可以解决呢？
怀疑是宽带限制，同上，重启alist并不会重新拨号或者改变宽带ip。
怀疑是123云盘限速，但是为什么会影响移动云盘呢
怀疑是alist的问题，那为什么会影响rclone呢？rclone是独立于alist的。

因为涉及敏感文件上传，所以暂时不方便提供alist日志。
因为不确定是不是alist的问题，所以不发到Issues而是先发到这里。

---------------------------------------

2024-8-16 更新：
上面的测试结果有误，最新测试是： 无论是123云盘还是移动云盘，上传大约100至300GB文件后，速度都会降低，重启后恢复正常。
另外，同一个网络，在另一台机子，通过onedrive客户端连续上传了600GB也限速了。
怀疑是运营商限速，但奇怪的时重启机器就可以恢复正常，不用重新拨号，不用等第二天。

**Comment by Simonzhang8** [comment]
~~也许是平台对cookie做了相应限制？（猜的）因为cookie重启后好像会自动清除~~
还有一种可能是你服务器性能的限制，传的太多导致占用过大而影响速度，重启Alist以后占用就没了速度也就回来了

**Reply by gdtv** [reply]
你指的是网盘的cookie？ 但是奇怪的是我在123云盘大量上传文件后，会导致移动云盘也限速，这个用cookie说不通吧。

**Reply by Simonzhang8** [reply]
> 你指的是网盘的cookie？ 但是奇怪的是我在123云盘大量上传文件后，会导致移动云盘也限速，这个用cookie说不通吧。

哦哦，要么还有一种可能是你服务器性能的限制，传的太多导致占用过大而影响速度，重启Alist以后占用就没了速度也就回来了 @gdtv

**Comment by LINJIAXIN-China** [comment]
我也有遇到这种现象，我最近是打算把夸克网盘的文件复制到天翼云盘（个人云），期间发现异常现象：
1、使用天翼云盘cookie进行登录，复制约200G文件后会无法继续执行，第二天恢复，排查为天翼云盘上传流量每日限制200G。
2、换用天翼云盘驱动，开启家庭转移，可绕过流量限制，但是不定期会任务假死在那里，不进行复制，任务仍显示在运行中，但就是无进度，查看alist网速占用几乎为0，alist几乎无占用性能。重启docker容器后可恢复。
